src/amo/components/Category/index.js

```diff
@@ -24,7 +18,10 @@ export class CategoryBase extends React.Component {
     category: PropTypes.object,
     dispatch: PropTypes.func.isRequired,
     errorHandler: PropTypes.object.isRequired,
+    filters: PropTypes.object,
     loading: PropTypes.boolean,
+    paginationQueryParams: PropTypes.object,
```

Comment:
Yeah, no worries, we can fix it up when porting the prop types to Flow. That proptype eslint module is way buggy. I usually just put a comment at the to to disable it!